src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1079,10 +1079,10 @@ export default class Aircraft {
                 break;
             case FLIGHT_MODES.WAITING:
                 runway = airport.getRunway(this.rwy_dep);
-                position = runway.position;
+                position = runway.position.relativePosition;
```

Comment:
using the same concept as above, you'd need another getter in the `RunwayModel` for `relativePosition` here